gateway/Makefile

```diff
@@ -1,20 +1,22 @@
 include ../config.mk
 
-.PHONY: build push deploy
+.PHONY: envoy-xds-config envoy-config deploy-no-xds deploy
 
-GATEWAY_IMAGE := $(DOCKER_PREFIX)/gateway:$(TOKEN)
 IP := $(shell kubectl get secret global-config --template={{.data.ip}} | base64 --decode)
+SHA := $(shell git rev-parse --short=12 HEAD)
 
-export NAMESPACE := default
+envoy-xds-config:
+	HAIL_DOMAIN=$(DOMAIN) python3 ../ci/ci/envoy.py gateway ${HAIL}/letsencrypt/subdomains.txt ${HAIL}/gateway/cds.yaml ${HAIL}/gateway/rds.yaml
+	kubectl -n default create configmap gateway-xds-config \
```

Comment:
Would it be too much work to add `.out` to the ends of the `rds.yaml` and `cds.yaml` files so it's clear these are temp files in the directory?

Reply:
Nope I can do that